bindata/network/multus/003-dhcp-daemon.yaml

```diff
@@ -0,0 +1,60 @@
+{{if .RenderDHCP}}
```

Comment:
So I actually meant to put this inside `multus.yaml`; rather than deploying an additional DaemonSet, with an additional Pod on each node, just stick the containers here into the existing multus daemonset, conditionally on `RenderDHCP` being set.

(This could be done as an optimization after the initial PR merges maybe though.)

Reply:
Hey Dan -- what's the best way to test for the presence of a certain named container in the unit tests? I saw there's a custom matcher that does it for specific `kind:` but, was having trouble testing for the presence of a specific container within a daemonset.